ivy_tests/test_ivy/test_frontends/test_torch/test_torch_nn_functional_dropout.py

```diff
@@ -0,0 +1,54 @@
+# global
+
+import numpy as np
+
+from hypothesis import strategies as st
+from ivy_tests.test_ivy.helpers import handle_frontend_test
+
+# local
+import ivy_tests.test_ivy.helpers as helpers
+
+
+@handle_frontend_test(
+    fn_tree="torch.nn.functional.dropout",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        min_value=0,
+        max_value=50,
+        allow_inf=False,
+        min_num_dims=1,
+        max_num_dims=1,
+        min_dim_size=2,
+    ),
+    prob=helpers.floats(min_value=0, max_value=0.9),
+    training=st.booleans(),
+    test_with_out=st.just(False),
+    test_inplace=st.booleans(),
+)
+def test_torch_dropout(
+    *,
+    dtype_and_input,
```

Comment:
this should be `dtype_and_x` as you are calculating `dtype_and_x` above in the decorator
your function is expecting `dtype_and_x` here.

Reply:
i made the changes you asked me for  please check the second commit @zaeemansari70